app/docs/dev/networking/non-mesh-traffic.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Non-mesh traffic
+---
+
+In its default setup, {{site.mesh_product_name}} allows any non-mesh traffic to pass Envoy without applying any policy. 
```

Comment:
Should we split this in 2? Incoming and Outgoing traffic?

I feel like it would be useful to cover incoming traffic too? Questions about this come up now and then.
Say something about impossible with mtls, link to permissive. Say that without transparent proxy people can bypass if you don't protect...